molo/core/models.py

```diff
@@ -444,10 +444,12 @@ class Tag(TranslatablePageMixin, Page):
     subpage_types = []
 
     feature_in_homepage = models.BooleanField(default=False)
+    feature_in_section = models.BooleanField(default=False)
```

Comment:
Yes we still need this. Because we may want to attach tags to Section Pages, but then only promote two of them to the section, and then later promote a different two. Or if we dont want this tag promoted in any section it's quicker to just untick this than remove it from all the section